packages/react-ui/src/widgets/TimeSeriesWidgetUI/TimeSeriesWidgetUI.js

```diff
@@ -345,6 +370,18 @@ function weeksCurrentDateRange(date) {
   return `Week of ${getMonday(date).toLocaleDateString()}`;
```

Comment:
It might be interesting to add (Monday) to the copy, as in US they also represent weeks starting from Sunday... what do you think @borja-munoz ?